pkg/auth/auth_handler.go

```diff
@@ -193,16 +205,20 @@ func (a *AuthHandler) AuthUserAuth(token string) (interface{}, error) {
 	return payload, nil
 }
 
-func (a *AuthHandler) storeAdminInPayload(payload *ocm.AuthPayload) error {
-	admin, err := a.isAdmin(payload.Username)
+func (a AuthHandler) storeAdminInPayload(payload *ocm.AuthPayload) {
+	payload.IsAdmin = funk.Contains(a.AdminUsers, payload.Username)
+}
+
+func (a *AuthHandler) storeReadOnlyAdminInPayload(payload *ocm.AuthPayload) error {
+	readOnlyAdmin, err := a.isReadOnlyAdmin(payload.Username)
 	if err != nil {
 		return err
 	}
-	payload.IsAdmin = admin
+	payload.IsReadOnlyAdmin = readOnlyAdmin
 	return nil
 }
 
```

Comment:
I think that payload should not have both `isReadOnlyAdmin` and `isAdmin`
We cannot have both of them at the same time